enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -110,6 +116,24 @@ Although cluster infra resources already do a best effort validation on the user
 a user could provide non-functional proxy settings/certs. This would be propagated to their Windows nodes and workloads,
 taking down existing application connectivity and preventing new Windows nodes from being bootstrapped.
 
+To enable global HTTP proxy by default on Windows nodes with cluster-wide proxy for all PowerShell's sessions, 
+the cluster administrator must create a default PowerShell profile script that reads the proxy environment variables 
+maintained by WMCO and populate the [DefaultWebProxy](https://learn.microsoft.com/en-us/dotnet/api/system.net.webrequest.defaultwebproxy) property
+
+The Powershell profile file location for all users is `$PROFILE.AllUsersCurrentHost` and, the proxy settings can be 
+updated with:
+```powershell
+    [System.Net.WebRequest]::DefaultWebProxy = New-Object System.Net.Webproxy("<PROXY_URL>")
+```
+where `PROXY_URL` is the URI of the cluster-wide proxy.
+
+Run the following commands in the Windows Node with cluster-wide proxy enabled to create a default profile for the 
+Powershell sessions that reads the proxy environment variables maintained by WMCO and populate the `DefaultWebProxy` property:
```

Comment:
We should mention the example commands below are only for HTTP_PROXY and this needs to be done for all proxy variables.